contracts/factory/src/msg.rs

```diff
@@ -17,12 +17,13 @@ pub struct InstantiateMsg {
     pub base_denom: String,
     pub core_params: CoreParams,
     pub staker_params: StakerParams,
-    pub puppeteer_params: PuppeteerParams,
+    pub fee_params: Option<FeeParams>,
 }
 
 #[cw_serde]
-pub struct PuppeteerParams {
-    pub timeout: u64,
+pub struct FeeParams {
+    pub fee: Decimal, // 0 - 1
+    pub fee_address: String,
```

Comment:
I see that you got rid of puppeteer_timeout, which is now set from remote_params. Then, why is staker_timeout still configurable in staker_params?